.gitmodules

```diff
@@ -2,8 +2,8 @@
   path = spack
   #url = https://github.com/spack/spack
   #branch = develop
-  url = https://github.com/NOAA-EMC/spack
-  branch = jcsda_emc_spack_stack
+  url = https://github.com/climbfuji/spack
+  branch = feature/merge_release_120_into_jcsda_emc_spack_stack
```

Comment:
Note to self. Revert after https://github.com/NOAA-EMC/spack/pull/208 was merged